packages/extras/src/lib/hooks/useGltfAnimations.ts

```diff
@@ -11,8 +11,8 @@ import { useFrame } from '@threlte/core'
  *
  * ```svelte
  * <script lang="ts">
- *   import { GLTF } from 'threlte'
- *   import { useGltfAnimations } from '../lib/useAnimations'
+ *   import { GLTF } from '@threlte/core'
```

Comment:
`GLTF` is from /extras, can be merged into one import as well

Reply:
done 👍🏻